backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/services/CoreFeaturesSwitch.java

```diff
@@ -93,6 +103,15 @@ public List<EStructuralFeature> caseFeature(Feature object) {
         return features;
     }
 
+    @Override
+    public List<EStructuralFeature> caseTransitionFeatureMembership(TransitionFeatureMembership object) {
+        var features = new ArrayList<EStructuralFeature>();
+        features.add(SysmlPackage.eINSTANCE.getElement_DeclaredName());
+        features.add(SysmlPackage.eINSTANCE.getTextualRepresentation_Language());
+        features.add(SysmlPackage.eINSTANCE.getTextualRepresentation_Language());
```

Comment:
Why adding the same feature twice? You probably wanted to add body instead of language

Reply:
I will remove this case as it seems a bad copy/paste and has nothing to do with this PR